README.md

```diff
@@ -1 +1,84 @@
-# MongoDB Atlas Cloudformation Resources
\ No newline at end of file
+# MongoDB Atlas Cloudformation Resources
+
+This is the repository for the Terraform MongoDB Atlas CloudFormation, which allows one to use Cloudformation with MongoDB's Database as a Service offering, Atlas.
+Learn more about Atlas at  [https://www.mongodb.com/cloud/atlas](https://www.mongodb.com/cloud/atlas)
```

Comment:
We may want to remove that mention of Terraform :)  Suggest the following:

This is the repository for the MongoDB Atlas Resource Provider for CloudFormation, which allows one to use CloudFormation to programmatically manage MongoDB Atlas, MongoDB's Database as a Service offering.
Learn more about Atlas at  [https://www.mongodb.com/cloud/atlas](https://www.mongodb.com/cloud/atlas)